frontend/test/metabase/lib/expressions/process.unit.spec.js

```diff
@@ -0,0 +1,30 @@
+import { processSource } from "metabase/lib/expressions/process";
+
+describe("metabase/lib/expressions/process", () => {
+  describe("processSource", () => {
```

Comment:
This new unit test proves that the functionality of `processSource` isn't affected by the change.